src/routes/pages/CartPage/CartPage.tsx

```diff
@@ -6,63 +6,53 @@ import ContainerLayout from '../../../components/common/ContainerLayout/Containe
 import CartListTitle from '../../../components/CartListTitle/CartListTitle';
 import CartItem from '../../../components/CartItem/CartItem';
 import CartPriceInfo from '../../../components/CartPriceInfo/CartPriceInfo';
-import OrderButton from '../../../components/OrderButton/OrderButton';
 import EmptyCart from '../../../components/EmptyCart/EmptyCart';
-import Toast from '../../../components/common/Toast/Toast';
-
 import { Logo } from '../../../assets';
 
-import useCartList from '../../../hooks/useCartList';
-import useSelect from '../../../hooks/useSelect';
 import LoadingSpinner from '../../../components/common/LoadingSpinner/LoadingSpinning';
-import { cartPrice } from '../../../utils/cartPrice';
-import { useToastContext } from '../../../context/ToastContext';
 import { useNavigate } from 'react-router';
 import { CartListStyle } from '../../../components/CartList/CartList.styles';
 import CartListHeader from '../../../components/CartList/CartList';
+import { TEXT } from '../../../constants/text';
+import { useCartContext } from '../../../context/CartContext';
+import Button from '../../../components/common/Button/Button';
+import Text from '../../../components/common/Text/Text';
 
 function CartPage() {
-  const cartList = useCartList();
-  const selectedList = useSelect(cartList.data);
-  const { isVisible } = useToastContext();
+  const cart = useCartContext();
   const navigate = useNavigate();
 
-  const totalPrice = cartPrice.totalPrice(
-    cartList.data,
-    selectedList.selectedItems
-  );
-
-  const selectedCartData = cartList.data.filter((item) =>
-    selectedList.selectedItems.includes(item.id)
+  const selectedCartData = cart.data.filter((item) =>
+    cart.selectedItems.includes(item.id)
   );
 
   const handleOrderButtonClick = () => {
-    navigate('/order-check', { state: { selectedCartData, totalPrice } });
+    navigate('/order-check');
   };
 
   const renderCartList = () => {
-    return cartList.data.length === 0 ? (
+    return cart.data.length === 0 ? (
       <EmptyCart />
     ) : (
       <>
         <CartListHeader
-          allSelected={selectedList.isAllSelected}
-          onAllSelectChange={selectedList.handleSelectAllItems}
+          allSelected={cart.isAllSelected}
+          onAllSelectChange={cart.selectAllItems}
         />
         <ul css={CartListStyle}>
-          {cartList.data.map((cartItem) => (
+          {cart.data.map((cartItem) => (
             <CartItem
               key={cartItem.id}
               cartItem={cartItem}
```

Comment:
```suggestion
              {...cartItem}
```
CartItem 입장에서 cartItem 이라는 이름으로 값을 받아오기보단, 이렇게 cartItem의 요소들이 값으로 해체되어 쓰이는게 좋지 않나 싶어요!